samples/vue/README.md

```diff
@@ -0,0 +1,52 @@
+# Vue
+
+This is an example how to use `xlsx-import` in Vue.
+It is made with [Vue CLI](https://cli.vuejs.org/)
+
+**LIB Version:** `2.3.3`
+
+## Scripts
+
+`npm run serve` - compiles and hot-reloads for development
+
+`npm run build` - compiles and minifies for production
+
+`npm run lint` - lints and fixes files
```

Comment:
Could I ask you to add script `start` which do exacly same logic like serve (or call `npm run serve`)?

 I'm asking about that, because all our samples has `start`.